### PR TITLE
Review collection actions

### PR DESCRIPTION
**Pull request by cashmonger**

### CODE REVIEW

File: CollectionActions.rb [hunk 1]
```diff
@@ -7,108 +7,110 @@
...
+    store_collection_materials(operations, location: location, role: 'input')
+    # show do
+    #   title 'Put Away the Following Items'
+    #   table material_storage_locations(operations, role: 'input',
+    #           location: location)
+    # end
   end
 
-  # Stores all output collections from all operations
+  # NOTE: Is there a reason these are seperate methods?
```
Comment (cashmonger) [inline CollectionActions.rb:23]: Is there a reason these are separate methods? Could just be one with the role specified when you call it.

File: CollectionActions.rb [hunk 2]
```diff
@@ -7,108 +7,110 @@
...
+    store_collection_materials(operations, location: location, role: 'output')
+    # show do
+    #   title 'Put Away the Following Items'
+     #  table material_storage_locations(operations, role: 'output',
+      #         location: location)
+   #  end
   end
 
-  # Store collections
+  # Instructions for technician on how and where to store materials
```
Comment (cashmonger) [inline CollectionActions.rb:37]: I've been trying to clarify in the notes which methods are instructions for the technician to work with literal items and which are instructions for Aquarium to work with virtual items.

File: CollectionActions.rb [hunk 3]
```diff
@@ -129,6 +131,7 @@ def store_items(obj_item, location: nil)
     end
   end
 
+  # NOTE: is this method meant to tell someone to trash an item or is it to make a table of items with the place to trash them? I am confused as it has a show block, but it returns a table?
```
Comment (cashmonger) [inline CollectionActions.rb:134]: As the note says -- is this a method to give instructions to the technician or to make a table? It has a show block, but then it returns a table.

File: CollectionActions.rb (hunk 1 above)
Comment (malloc3) [reply CollectionActions.rb:23]: These could be the same method with role called yes

File: CollectionActions.rb (hunk 2 above)
Comment (malloc3) [reply CollectionActions.rb:37]: Ahh thats a good idea.  In the future I will do that

File: CollectionActions.rb (hunk 3 above)
Comment (malloc3) [reply CollectionActions.rb:134]: It is to give instructions.   It wasn't intended to return a table.  I think thats just from the inherent ruby thing where it returns what ever object is the last line in the method...    Not sure, not supposed to return anything just display the table with instructions to throw stuff away.

**Review by malloc3 — APPROVED** [review]

**Pull request merged**